common/app/common/LinkTo.scala

```diff
@@ -29,42 +29,42 @@ trait LinkTo extends Logging {
   def apply(html: Html)(implicit request: RequestHeader): String = this(html.toString(), Edition(request))
   def apply(link: String)(implicit request: RequestHeader): String = this(link, Edition(request))
 
-  def apply(url: String, edition: Edition)(implicit request: RequestHeader): String = {
-    val processedUrl: String = processUrl(url, edition).url
-    handleQueryStrings(processedUrl)
-  }
+  def apply(url: String, edition: Edition)(implicit request: RequestHeader): String =
```

Comment:
thanks for the comment and it's definitely not perfect, next time I'm tweaking this code I'll try to do that